### ISSUE TITLE
Lol, You deleted the your character certificate ?

### ISSUE THREAD

**Issue by creatiosoftrishabh** [shohidulbari/redis-streams-client#1]
Let me post everything in your country forum. Wait for me.

**Comment by shohidulbari** [comment]
https://github.com/shohidulbari/callfor/issues/1

**Comment by shohidulbari** [comment]
why will I delete that? Am I a coward?

**Comment by creatiosoftrishabh** [comment]
No, You are not coward. You are someone which I can't say here. Keep it.

**Comment by shohidulbari** [comment]
Well, Actually I don't understand why you think everything emotionally. I worked on your project and when I left that project was in a good state. If you can't manage your project, from which point you are blaming me? I offered you documentation, onboarding, and help. What else I could offer! did you expect that I would work for you forever?

**Comment by creatiosoftrishabh** [comment]
No I never expect you to work for us forever. But you guys cheated us. I know who placed you as multiple time we discussed about that and some kind of confirmation also. Issue is that if you clearly communicate on last call to me and Nishank that you don't want to work with us, it was fine. But you created a story on gun point thats a unethical.  Monthly contract is monthly contract and never can be terminated b/w the week.

I clearly asked you many time ( look and ask to your self) that if you are not happy to working with us its fine, just tell and move on but you always said you are happy and want to work with us. Behind all this, you guys were creating different story this is unethical.

**Comment by shohidulbari** [comment]
There was no story. Now I am getting time for my higher study. Everybody has their own ethics. I have seen your ethics also which is unethical for me. You can't blame others blindly. There was no gunpoint, I proposed a proposal. You could have discussed that with me but you tried to act forcefully.

**Comment by creatiosoftrishabh** [comment]
Ok, If there is no story then why in this recording you didn't tell us you have a joining on 15th. My forcefully only came after your story came into the picture. Listen this recording. You created 15th story because you agreed to join other company on same date. 

https://playmooglec-public.s3.us-west-2.amazonaws.com/discussion.m4a

**Comment by creatiosoftrishabh** [comment]
We never expect you from to work for us forever as no buddy work. But while you were into monthly contract (We were paying everything month not each week) and 1 month notice period confirmation ( On Skype) and suddenly planting bogus story is fall into  cheated category.

**Comment by shohidulbari** [comment]
Was it required to tell you? There were many reasons to leave your project, I told you one. As I was working as a free-lancer and there was no written agreement on it, I think this is my personal matter what will I do. You could have any time terminate my service also. Anyway what I can remember, I offered my help and support till the project is finished. But you tried forcefully that's why I stopped working. No problem, you can tell anyone about this. I don't care about it. Nobody will care about this. Because there is no point. But I feel sorry for you that you can't manage your project when one developer leaves. It is your responsibility. Doesn't mean to blame anybody for it.
